packages/browser/test/e2e/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "sentry-browser-integration-tests",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "engines": {
+    "node": ">=16"
+  },
+  "private": true,
+  "scripts": {
+    "install-browsers": "playwright install --with-deps",
+    "pretest": "yarn install && yarn install-browsers",
```

Comment:
We should cache the browser binaries instead of reinstalling in every run.
I shall have an example of that on Monday.

In short, it means adding `~/.cache/ms-playwright/` to `CACHED_DEPENDENCY_PATHS`, or using `PLAYWRIGHT_BROWSERS_PATH` to tell Playwright to use a different path (and cache that path).